src/applications/edu-benefits/1990/containers/IntroductionPage.jsx

```diff
@@ -154,7 +148,7 @@ export class IntroductionPage extends React.Component {
 }
 
 const mapStateToProps = state => ({
-  shouldEduBenefits1990WizardShow: showEduBenefits1990Wizard(state),
+  showWizard: showEduBenefits1990Wizard(state),
```

Comment:
It's a little confusing to me when `status` in the state is different from `showWizard`/`showEduBenefits1990Wizard`. Is that `showEduBenefits1990Wizard` pulls the value out of the Redux store, which may be out of sync with this component state?

Reply:
i agree there should be a default state set for this component ill update it thanks